frontend/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:20-alpine AS builder
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+COPY ./node_modules ./node_modules
+COPY . .
+
+RUN npm install --production
+RUN npm run build
+
+FROM node:20-alpine AS runner
+WORKDIR /app
+
+ENV NODE_ENV=production
+
+COPY --from=builder --chown=nextjs:nodejs /app/.next ./.next
+COPY --from=builder /app/node_modules ./node_modules
```

Comment:
Ei kopioida tätä. Vaan asennetaan node_modules:it komennolla `RUN npm ci --omit=dev` package.json kopioinnin jälkeen